CMakeLists.txt

```diff
@@ -11,11 +11,19 @@ set(CUKE_ENABLE_EXAMPLES OFF CACHE BOOL "Enable the examples")
 set(GMOCK_DIR "" CACHE STRING "Google Mock framework sources path (otherwise downloaded)")
 set(GMOCK_VER "1.7.0" CACHE STRING "Google Mock framework version to be used")
 set(VERBOSE OFF CACHE BOOL "Verbose output")
+set(APPVEYOR OFF CACHE BOOL "Is this appveyor CI build")
```

Comment:
Don't like polluting the `CMakeLists.txt` with anything related to the CI build. I'd rather have AppVeyor specifying the right flags.